<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createlineDialog</class>
 <widget class="QDialog" name="createlineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>281</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>243</width>
    <height>281</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>243</width>
    <height>281</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Start X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editStartX"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Start Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editStartY"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>End X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editEndX"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>End Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editEndY"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnEnterData">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Cancle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
